core/templates/pages/about-page/about-page.module.ts

```diff
@@ -82,3 +87,11 @@ const downgradedModule = downgradeModule(bootstrapFn);
 declare var angular: ng.IAngularStatic;
 
 angular.module('oppia').requires.push(downgradedModule);
+
+angular.module('oppia').directive(
+  // This directive is the downgraded version of the Angular component to
+  // bootstrap the Angular 8.
+  'oppiaAngularRoot',
+  downgradeComponent({
+    component: OppiaAngularRootComponent
```

Comment:
Are we going to do this in all the directives which are going to use oppia-root?

Reply:
No directive will use oppia-root! This is the root of our application. Inside this, we will have our application. So this will only be done only once in all modules.